HW03/Mikhail_Sauchuk/abstract_observer.rb

```diff
@@ -0,0 +1,29 @@
+# abstract subject class
+class AbstractObserver
+  attr_accessor :homeworks, :students, :surname
```

Comment:
do you have specific reasons for using `attr_accessor` instead of `attr_reader` here?
```suggestion
  attr_reader :homeworks, :students, :surname
  
```